repositories.bzl

```diff
@@ -0,0 +1,62 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def com_google_protoc_java_resource_names_plugin_repositories(
+    omit_com_google_protobuf = False,
+    omit_pypi_py_yaml = False,
+    omit_pypi_pystache = False,
+    omit_pypi_six = False):
+    if not omit_com_google_protobuf:
+        com_google_protobuf()
+    if not omit_pypi_py_yaml:
+        pypi_py_yaml()
+    if not omit_pypi_pystache:
+        pypi_pystache()
+    if not omit_pypi_six :
+        pypi_six()
+
+def com_google_protobuf():
+    native.git_repository(
+        name = "com_google_protobuf",
+        remote = "https://github.com/google/protobuf.git",
+        tag = "v3.6.0",
+    )
+
+def pypi_py_yaml():
+    http_archive(
+        name = "pypi_py_yaml",
+        url = ("https://files.pythonhosted.org/packages/4a/85/db5a2df477072b2902b0eb892feb37d88ac635d36245a72a6a69b23b383a/PyYAML-3.12.tar.gz"),
+        strip_prefix = "PyYAML-3.12/lib/yaml",
+        build_file_content = """
```

Comment:
This is quite ugly... is there a way to make a var out of this, so the `def` looks a little cleaner?

Reply:
Yes, it can be done, but the current way is the suggested way of doing it in bazel ([best practices](https://docs.bazel.build/versions/master/best-practices.html#custom-build-files)). We can also have separate build files for this and provide `build_file` parameter instead of `build_file_content` in `http_archive()` rule, but that would be against best practices and, "heavier" than it should be (at long last it is just pulling a dependency, so having it as single command (not composed from variables and/or external files) should make sense).

Reply:
Oh I had in mind something much lighter - just a variable above the definition of `pypi_py_yaml()`. Having the `"""` text within the rule makes indentation funny.